src/components/SelectorInput/selector-input-color-palette.js

```diff
@@ -0,0 +1,10 @@
+export const BACKGROUND = '#2f5e5a';
```

Comment:
Do we already have / plan to have a global palette? I believe the same colours will used in other components (like Round Name, Pearl Thread etc)

Reply:
If we have time for that, I'd also reorganise the palette a bit into colors and their semantic assignments. That's if we really have a fixed (minimal number of colors) color palette, which is generally a good thing.

Smth. like
```js
const WHITE = '#fff' // is not recommended btw
export const GENERIC_TEXT = WHITE // only semantic constants are exported, pure colors can't be used without a meaningful UX role.

```

Reply:
I think it will be easier to implement components in this way and later extract same colors to color palette.

```javascript
const WHITE = '#fff' // is not recommended btw
```
Agree, I prefer smth like `BRAND_PRIMARY`, `BRAND_ACCENT` etc.

Reply:
> I think it will be easier to implement components in this way and later extract same colors to color palette.
Sure, let's just maybe create a task/issue for that?

Concerning white, it's just that pure white is not recommended as such, it's too "technical" a color — shades of white, warm or cool, should be preferred.

`BRAND_PRIMARY` and such would be semantic names.

Reply:
> let's just maybe create a task/issue for that?

#20

Reply:
👍